polar-core/src/error.rs

```diff
@@ -86,13 +85,21 @@ impl PolarError {
         }
 
         // Augment ResourceBlock errors with relevant snippets of parsed Polar policy.
-        if let ErrorKind::Parse(ParseError::ResourceBlock {
+        if let ErrorKind::Validation(ValidationError::ResourceBlock {
             ref mut msg,
             ref ranges,
-            ..
+            ref loc,
         }) = self.kind
         {
             if let Some(source) = source {
```

Comment:
Is `source` ever set on the context for resource block errors? Where's that happening? 

Related to my other comment about carrying through `term` instead of `loc`. I'm wondering if we would even know which file the error comes from if we throw away the term

Reply:
82dd969 sets context & carries around the relevant term for its lexical info